openwisp_monitoring/device/static/monitoring/css/daterangepicker.css

```diff
@@ -0,0 +1,432 @@
+  border: 1px solid #eee;
+  padding: 2px;
+  outline: 0;
+  font-size: 12px;
+}
+.daterangepicker .calendar-time {
+  text-align: center;
+  margin: 4px auto 0 auto;
+  line-height: 30px;
+  position: relative;
+}
+.daterangepicker .calendar-time select.disabled {
+  color: #ccc;
+  cursor: not-allowed;
+}
+.daterangepicker .drp-buttons {
+  clear: both;
+  text-align: right;
+  padding: 8px;
+  border-top: 1px solid #ddd;
+  display: none;
+  line-height: 12px;
+  vertical-align: middle;
+}
+.daterangepicker .drp-selected {
+  display: inline-block;
+  font-size: 12px;
+  padding-right: 8px;
+}
+.daterangepicker .drp-buttons .btn {
+  margin-left: 8px;
+  font-size: 12px;
+  font-weight: bold;
+  padding: 4px 8px;
+}
+.daterangepicker.show-ranges.single.rtl .drp-calendar.left {
+  border-right: 1px solid #ddd;
+}
+.daterangepicker.show-ranges.single.ltr .drp-calendar.left {
+  border-left: 1px solid #ddd;
+}
+.daterangepicker.show-ranges.rtl .drp-calendar.right {
+  border-right: 1px solid #ddd;
+}
+.daterangepicker.show-ranges.ltr .drp-calendar.left {
+  border-left: 1px solid #ddd;
+}
+.daterangepicker .ranges {
+  float: none;
+  text-align: left;
+  margin: 0;
+}
+.daterangepicker.show-calendar .ranges {
+  margin-top: 8px;
+}
+.daterangepicker .ranges ul {
+  list-style: none;
+  margin: 0 auto;
+  padding: 0;
+  width: 100%;
+}
+.daterangepicker .ranges li {
+  font-size: 12px;
+  padding: 8px 12px;
+  cursor: pointer;
+}
+.daterangepicker .ranges li:hover {
+  background-color: #eee;
+}
+.daterangepicker .ranges li.active {
+  background-color: #08c;
+  color: #fff;
+}
+.daterangepicker {
+  width: auto;
+}
+.daterangepicker .ranges ul {
+  width: 140px;
+}
+.daterangepicker.single .ranges ul {
+  width: 100%;
+}
+.daterangepicker.single .ranges,
+.daterangepicker.single .drp-calendar {
+  float: left;
+}
+.daterangepicker {
+  direction: ltr;
+  text-align: left;
+}
+.daterangepicker .drp-calendar.left {
+  clear: left;
+  margin-right: 0;
+}
+.daterangepicker .drp-calendar.left .calendar-table {
+  border-right: none;
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+}
+.daterangepicker .drp-calendar.right {
+  margin-left: 0;
+}
+.daterangepicker .drp-calendar.right .calendar-table {
+  border-left: none;
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+}
+.daterangepicker .drp-calendar.left .calendar-table {
+  padding-right: 8px;
+}
+.daterangepicker .ranges,
+.daterangepicker .drp-calendar {
+  float: left;
+}
+.daterangepicker .drp-calendar.left {
+  clear: none;
+}
+.daterangepicker .ranges li {
+  list-style: none;
+}
+.daterangepicker .ranges ul li.active {
+  background-color: #fce5e5;
+  color: #df5d43;
+}
+.daterangepicker .ranges ul li:hover {
+  background-color: #fce5e5;
+  color: #e4715a;
+}
+.daterangepicker .drp-buttons button.btn {
+  margin: 0 2px;
+  color: #fff;
+  background-color: #666;
+  padding: 5px 15px;
+  text-decoration: none;
+  border-radius: 3px;
+  cursor: pointer;
+}
```

Comment:
If we remove the border from these buttons, they will look like other buttons in OpenWISP 

![Screenshot from 2022-10-25 16-35-24](https://user-images.githubusercontent.com/32094433/197757709-d5d9e25c-4e3d-43ee-a745-e040faaac9fc.png)

```css
border: 0;
padding: 0.5rem 1rem;
```

Reply:
I'll update that, thanks :+1: